tests/python/relax/test_tvmscript_parser.py

```diff
@@ -184,6 +184,39 @@ def foo(x: R.Tensor((128, 128), "float32")) -> R.Tensor((128, 128), "float32"):
     _check(TestModule, bb.get())
 
 
+def test_emit_te_primfunc_attrs():
+    @I.ir_module
+    class TestModule:
+        @T.prim_func
+        def plus_one(
+            x: T.Buffer((T.int64(128), T.int64(128)), "float32"),
+            y: T.Buffer((T.int64(128), T.int64(128)), "float32"),
+        ):
+            T.func_attr({"some_attr": "foo", "another_attr": True, "tir.noalias": True})
+            for i, j in T.grid(T.int64(128), T.int64(128)):
+                with T.block():
+                    vi, vj = T.axis.remap("SS", [i, j])
+                    y[vi, vj] = x[vi, vj] + 1.0
+
+        @R.function
+        def foo(x: R.Tensor((128, 128), "float32")) -> R.Tensor((128, 128), "float32"):
+            # TODO(Siyuan): Need to change to `TestModule.tir_func`
```

Comment:
remove this todo here, it was tracked by somewhere else

Reply:
done, thanks!